go.sum

```diff
@@ -10,11 +10,16 @@ github.com/antchfx/xpath v1.1.2 h1:YziPrtM0gEJBnhdUGxYcIVYXZ8FXbtbovxOi+UW/yWQ=
 github.com/antchfx/xpath v1.1.2/go.mod h1:Yee4kTMuNiPYJ7nSNorELQMr1J33uOpXDMByNYhvtNk=
 github.com/c9s/goprocinfo v0.0.0-20191125144613-4acdd056c72d h1:MQGrhPHSxg08x+LKgQTOnnjfXt+p+128WCECqAYXJsU=
 github.com/c9s/goprocinfo v0.0.0-20191125144613-4acdd056c72d/go.mod h1:uEyr4WpAH4hio6LFriaPkL938XnrvLpNPmQHBdrmbIE=
+github.com/cenkalti/backoff/v4 v4.0.0 h1:6VeaLF9aI+MAUQ95106HwWzYZgJJpZ4stumjj6RFYAU=
```

Comment:
go.sum and go.mod: how are these files created? why they are needed?

Reply:
So the go.sum and go.mod files are automatically created by golang when the code is built. They are used to manage dependencies needed by the code. Go.mod lists and tracks dependencies including versions for a module. Go.sum authenticates the code that is being loaded and stores a hash of expected files. I have fixed a merge issue and these no longer show up as changes.